client/my-sites/site-settings/form-general.jsx

```diff
@@ -347,6 +347,23 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	publicFormRadio = ( checked, onChange ) => {
```

Comment:
Did those changes slip from the other PR?

Reply:
Strange, i thought i was branching off of trunk but I guess not 🤡 

I did an interactive rebase to solve it.